### ISSUE TITLE
Problem: reproduce the result,  SVHN to MNIST.

### ISSUE THREAD

**Issue by lavonne1213** [haeusser/learning_by_association#4]
Hello, I was trying to reproduce the result on the paper too,  SVHN to MNIST. First I download your code, chaning code 'target_dataset', None  into  "target_dataset', 'mnist3'   in train.py, then run train.py, after 100000 traning steps finished, I run the eval.py, but I can't get the Accuracy value. Please see details of result in attachment.

[result.txt](https://github.com/haeusser/learning_by_association/files/1244293/result.txt)

**Comment by haeusser** [comment]
Hi @lavonne1213 ,

you did everything correctly. The evaluation loop logs the results to the logdir. The idea is to use [TensorBoard](https://www.tensorflow.org/get_started/summaries_and_tensorboard) to monitor the training progress.

Cheers,
Philip

**Comment by 4Statistics** [comment]
When I use TensorBoard,where to find the acuracy value?what's its title or name?Thank you .
The title of the first graph  is "Learning rate",where is the accuracy?.Thank you .

**Comment by 4Statistics** [comment]
Very sorry to be annoying! But I really don't know how to run your code normally.
